test/factories/plan.rb

```diff
@@ -27,7 +27,15 @@
   end
 
   factory(:application_plan_without_rules, parent: :plan, class: ApplicationPlan) do
+    transient do
+      default { false }
+    end
```

Comment:
why is this?

Reply:
transient is used to add custom arguments to the factories. `cinstance` model does not have a `default` argument in the construction. This transient is used to make the plan, e.g. https://github.com/3scale/porta/blob/9598ff563fb4d383349cfcdf70118909dae62ceb/test/factories/plan.rb#L33